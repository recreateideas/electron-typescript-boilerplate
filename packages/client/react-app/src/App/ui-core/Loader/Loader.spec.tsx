import { Loader } from './Loader';
import { TestProvider } from '../../../tests';

const { mountWithProvider } = TestProvider({});

describe('<Loader/>', () => {
    it('should render', () => {
        const root = mountWithProvider(<Loader />);

        expect(root).toMatchSnapshot();
    });
});
