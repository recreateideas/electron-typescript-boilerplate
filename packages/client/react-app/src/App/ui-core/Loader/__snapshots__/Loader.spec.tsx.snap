// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> should render 1`] = `
<div
  className="sc-bdnxRM ivVdKc loader undefined"
>
  <div
    className="react-spinner-material"
    style={
      Object {
        "borderColor": "orange",
        "borderWidth": 5,
        "height": 40,
        "width": 40,
      }
    }
  />
</div>
`;
